docs/import/manage_layers.js

```diff
@@ -321,13 +315,6 @@ function populateStateAndDisasterAssetPickers(disaster) {
     });
     promises.push(disasterDone);
```

Comment:
Slightly simpler to return disasterDone here and either Promise.resolve() in the if block or allow this method to return nothing in the case that we didn't need to call getDisasterAssetsFromEe